### DISCUSSION TITLE
Send Later not sending any mail

### DISCUSSION THREAD

**Discussion by epsso** [Extended-Thunder/send-later#614] [General]
starting today, send later is not sending any mail.  I have TB 102.14.0 (64-bit) and send-later 10.1.0.  It is set to run every minute and it gives me the following.

Sending message 517beb1f-47db-018e-8be9-ee71608313a6@spacegrant.org. background.js:654:16
Sending message 3c921314-6611-f474-7768-711478cda6e7@epss.net. background.js:654:16
Sending message c6a28e79-40ff-908d-62b8-3d672788ea0b@epss.net. background.js:654:16
Sending message bf3ddb5b-72cb-75f1-e5ce-a281a0854f75@epss.net. background.js:654:16
Sending message 547e4996-e458-791c-b464-b7b8bcf29735@epss.net. background.js:654:16
Sending message 239e8de3-0419-b8bc-47e8-9e7d5d09f57d@epss.net. background.js:654:16
Sending message bc9f2c89-e32a-2acb-2ca4-b752fd080e9f@epss.net. background.js:654:16
Sending message 760e7413-68b9-92cf-9010-d4725ffeb9d7@epss.net. background.js:654:16
Sending message 1ff24073-ee42-799f-431c-df126fe7d9c1@epss.net. background.js:654:16
Sending message 94197cf0-035f-479a-fb87-ee357c66a802@epss.net. background.js:654:16
`<unavailable>`

The `<unavailable>` is red and shows up as an error but when expanded it is empty.

**Comment by epsso** [comment]
Earlier today I had to create OpenPGP keys for each email account that I wanted to use send-later from because that was the only way to uncheck the "Store draft messages in encrypted format" checkbox, which was blocking send-later. Here are my end-to-end encryption settings.

![E2E settings](https://github.com/Extended-Thunder/send-later/assets/26418430/fa4755db-7a01-4607-b1dc-642deb46e1ff)

**Comment by jikamens** [comment]
The encryption issue and the not sending on TB102 issue are two separate issues.

Both will be fixed in a release later today.

**Comment by jikamens** [comment]
I have released version 10.1.1 of Send Later which fixes all of the issues people have encountered today. The full list of fixes in this version is viewable on its [GitHub release page][rp].

The new release has been uploaded to addons.thunderbird.net and is waiting for the moderators there to approve it. If you would rather not wait for them, you can get it as follows:

* Download [this file][xpi] and save it to your local computer. If you're reading this in Firefox, make sure to right-click on the link and select "Save Link As..." rather than just clicking on it, or Firefox will try to install it as a Firefox add-on.
* Open the Add-ons page in Thunderbird (Tools > Add-ons and Themes).
* Click the gear icon at the top.
* Select "Install Add-on from File...".
* Browse to the file you saved and select it.
* Click through the dialogs to confirm installation.
* You can delete the downloaded file once you've installed it.

If you don't want to do this you can wait until the moderators approve it, though I can't say whether that will take hours or days.

I'm sorry about these bugs and I thank you for your patience as I've fixed them as rapidly as I could.

[rp]: https://github.com/Extended-Thunder/send-later/releases/tag/v10.1.1
[xpi]: https://github.com/Extended-Thunder/send-later/releases/download/v10.1.1/send_later.xpi
